Automattic/wp-calypso
# Hosting metrics: PHP and Webserver log table improvements

- Closes https://github.com/Automattic/dotcom-forge/issues/3392

## Proposed Changes

Minor style improvements on `/site-monitoring/${siteSlug}/php` and `/site-monitoring/${siteSlug}/web`.

* Remove border bottom when cell is expanded.
* Add flexible columns (Webserver `request_url` and PHP `message` by fixing the width in the rest.
* Remove cells' padding on mobile.
* Display full width the flexible column on mobile.
* Display chevron top right on mobile.
* Display loading skeleton only when changing tabs and not in pagination or auto-refresh.
* Change the color of the table from gray-80 to gray-100.
* Make the badges homogenous and 20px height.
* Display the correct label on tab nav for mobile, instead of the tab key.

## Testing Instructions



* On an atomic site, navigate to `/site-monitoring/${siteSlug}/php`
* Resize the window bigger and smaller
* Observe the message column displays some ellipsis and the column resizes.
* Click on Webserver
* Observe the same behaviour for `request_url` column
* Try the mobile versions and observe the new designs on the chevron, bottom border and full width of the flexible column.


## Screencast

**Flexible columns**

https://github.com/Automattic/wp-calypso/assets/779993/d6b4f8da-2fe3-4ae4-bfde-f6d7ccece7b5


## Screenshots

|| **BEFORE** | **AFTER** |
|---|---|---|
|PHP Mobile|<img width="366" alt="before-mobile-php" src="https://github.com/Automattic/wp-calypso/assets/779993/bbbe44f8-e38f-4212-b663-058af764fba3"> | <img width="366" alt="after-mobile-php" src="https://github.com/Automattic/wp-calypso/assets/779993/05f7480a-575a-4aaa-ba0f-0001dba4a1ea"> |
|Web Mobile|  <img width="364" alt="before-mobile-web" src="https://github.com/Automattic/wp-calypso/assets/779993/bdc8fa5d-2366-42b1-bcfb-3748584a8fbd"> | <img width="362" alt="after-mobile-web" src="https://github.com/Automattic/wp-calypso/assets/779993/83250982-8560-41e5-b2bc-efdf0979ca94"> |
| PHP Desktop | <img width="1612" alt="before-desktop-php-" src="https://github.com/Automattic/wp-calypso/assets/779993/1e3a01a3-b254-4b13-9ce5-b54f202a4208"> | <img width="1612" alt="after-desktop-php" src="https://github.com/Automattic/wp-calypso/assets/779993/567021f8-3339-41cc-9ecc-5918a16e6c9c"> |
| Web Desktop | <img width="1614" alt="before-desktop-web" src="https://github.com/Automattic/wp-calypso/assets/779993/ee0a766f-8fe2-40b7-9fb4-c28c0cfb9a71"> | <img width="1619" alt="after-desktop-web" src="https://github.com/Automattic/wp-calypso/assets/779993/6d9d8b90-d0c4-432c-993e-cd8beffa49f6"> |


## Pre-merge Checklist
